packages/react-kitchen-sink/src/schema/__tests__/ZodSchemaError.test.ts

```
@@ -0,0 +1,25 @@
+import { type QueryDocumentSnapshot } from '@firebase/firestore'
+import { type ErrorEvent } from '@sentry/react'
```

ESLint

[error] Unable to resolve path to module '@sentry/react'.

(import-x/no-unresolved)